app/views/content/life-as-a-teacher/chemistry/_adviser-promo-chemistry.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
```

Comment:
This is fine for now but I think we need to have a better way of doing these promos using dynamic content, so we are not creating a new one for each subject! Let's discuss separately